app/src/main/java/co/nimblehq/alpine/sample/CameraCaptureActivity.kt

```diff
@@ -99,14 +110,50 @@ class CameraCaptureActivity : ComponentActivity() {
         imageCapture = ImageCapture.Builder()
             .setCaptureMode(ImageCapture.CAPTURE_MODE_MINIMIZE_LATENCY)
             .build()
+        imageAnalysis = getImageAnalysis()
 
         camera = cameraProvider.bindToLifecycle(
-            this, cameraSelector, preview, imageCapture
+            this, cameraSelector, preview, imageCapture, imageAnalysis
         )
         // Attach the viewFinder's surface provider to Preview use case
         preview?.setSurfaceProvider(binding.pvCameraCaptureViewFinder.surfaceProvider)
     }
 
+    private fun getImageAnalysis(): ImageAnalysis {
+        val imageAnalysis = ImageAnalysis.Builder()
+            .setOutputImageFormat(OUTPUT_IMAGE_FORMAT_YUV_420_888)
+            .setTargetResolution(Size(1280, 720))
+            .setBackpressureStrategy(STRATEGY_KEEP_ONLY_LATEST)
+            .build()
+        imageAnalysis.setAnalyzer(cameraExecutor) { imageProxy: ImageProxy ->
+            mrzProcessor.processImage(imageProxy.toCameraImage(), object : MrzProcessorResultListener {
+                override fun onSuccess(mrzInfo: MrzInfo) {
+                    imageAnalysis.clearAnalyzer()
+                    imageProxy.close()
+                    NfcScanningActivity.start(this@CameraCaptureActivity, mrzInfo)
+                }
+
+                override fun onError(e: MrzProcessorException) = imageProxy.close()
```

Comment:
@ryan-conway Regarding your comment [here](https://github.com/nimblehq/alpine/pull/33#discussion_r982125445) Is this the same 🤔 Should we call `imageAnalysis.clearAnalyzer()` so the user should consider to manually capturing instead

Reply:
@Wadeewee Probably best if we don't call it 😄  `ImageAnalysis.clearAnalyzer` should be called when we want to _stop_ analyzing images. If we call it in `onError` then the `ImageAnalysis` will just scan the first frame, see that there is no MRZ detected so `onError` will be called once, then it will remove the analyzer and will not scan anymore 😅

Reply:
Ah, I see maybe I confused 🙈